.github/workflows/build-build-tools-image.yml

```diff
@@ -99,5 +103,5 @@ jobs:
       - name: Create multi-arch image
         run: |
           docker buildx imagetools create -t neondatabase/build-tools:${IMAGE_TAG} \
-                                             neondatabase/build-tools:${IMAGE_TAG}-x64 \
-                                             neondatabase/build-tools:${IMAGE_TAG}-arm64
+                                             neondatabase/build-tools:${IMAGE_TAG}-${{ matrix.os }}-x64 \
+                                             neondatabase/build-tools:${IMAGE_TAG}-${{ matrix.os }}-arm64
```

Comment:
It seems arch and os are swapped here

Reply:
this one is confusing, should we have `matrix` for [this](https://github.com/neondatabase/neon/blob/e0dadd50fa82c96047264eedd488c06fc5c04890/.github/workflows/build-build-tools-image.yml#L105) CI stage as well?